### ISSUE TITLE
Week 1 discussion: convert `tmap` code to match package version 3.3.4

### ISSUE THREAD

**Issue by julietcohen** [EDS-223-Geospatial/EDS-223-Geospatial.github.io#66]
- The answer key includes creating a `tmap` that uses code for a dev version of `tmap`. Convert the code to instead match `tmap` version 3.3.4, which is the latest version you can currently download from CRAN using simply `install.packages('tmap')`
  - the code chunk provided for [this map](https://r-tmap.github.io/tmap-book/nutshell.html#fig:rmap1) runs with version 3.3.4 
- ~remove duplicate data import chunk~

**Comment by julietcohen** [comment]
Related to issue #58 to create HW answer keys. I can take over this week's answer key and make sure Ale approves

**Comment by julietcohen** [comment]
I now realize that what I thought was a duplicate data import chunk is just your difference in filepaths between the data dir we use for the website repo and the data dir the students have for their repo

**Comment by julietcohen** [comment]
With [this PR](https://github.com/EDS-223-Geospatial/EDS-223-Geospatial.github.io/pull/70), I added the static and interactive Easter Island maps for `tmap` version 3.3.4 to week 1 discussion and rebuilt the website. I didn't delete Ale's code for the same maps with `tmap` version 3.9, I just commented it out. It would look cleaner to remove that code completely, so Ale can do that when she makes the switch to 3.3.4.
